src/agent/src/main.rs

```diff
@@ -110,21 +110,20 @@ fn announce(logger: &Logger, config: &AgentConfig) {
 
 fn set_fd_close_exec(fd: RawFd) -> Result<RawFd> {
     if let Err(e) = fcntl::fcntl(fd, FcntlArg::F_SETFD(FdFlag::FD_CLOEXEC)) {
-        return Err(anyhow!("failed to set fd: {} as close-on-exec: {}", fd, e));
+        bail!("failed to set fd: {} as close-on-exec: {}", fd, e);
```

Comment:
This is simple, but I'm confused if should we replace all `return Err(anyhow!())` to `bail!` in all the rust files.

Reply:
Until someone objections, I'll send another PR later to do tree-wide changes for this.

Reply:
I agree with @liubin, I think we should replace all `return Err(anyhow!)` to `bail!`, or we don't change any of it.

Reply:
FYI: I have remove this function `set_fd_close_exec ` in another pr: https://github.com/kata-containers/kata-containers/pull/1648

Reply:
> I agree with @liubin, I think we should replace all `return Err(anyhow!)` to `bail!`, or we don't change any of it.

Okay, I'll resubmit to defer this change for this PR.

Reply:
> FYI: I have removed this function `set_fd_close_exec ` in another pr: #1648

Okay, I'll rebase on it once your PR get merged (hopefully tomorrow).

Reply:
thanks a lot